spring/spring-data/src/test/java/io/quarkus/ts/spring/data/rest/JpaRepositoryRestResourceIT.java

```diff
@@ -0,0 +1,154 @@
+package io.quarkus.ts.spring.data.rest;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.ts.spring.data.AbstractDbIT;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class JpaRepositoryRestResourceIT extends AbstractDbIT {
```

Comment:
We have three almost identical test classes with requested URI(and a couple of methods) as their only difference. I suggest to move all common code into AbstractDBIT and left only getEndpoint method in the inherited classes

Reply:
Common code moved to new abstract ancestors.